lib/media/media_source_engine.js

```diff
@@ -1673,7 +1702,6 @@ shaka.media.MediaSourceEngine = class {
       }
       await Promise.all(cleanup);
       this.transmuxers_ = {};
-      this.queues_ = {};
```

Comment:
Why was this removed? I assume so that operations can continue being added, but what about the case where we are still set to RESET mode?

Reply:
In the event that we are configured in reset mode, the operations are overwritten without causing any side effects.